djstripe/urls.py

```diff
@@ -24,7 +24,7 @@
 
 urlpatterns = [
     # Webhook
-    url(
+    re_path(
```

Comment:
Was this left as `re_path` on purpose? If so my breaking change comment is moot.

Reply:
yes :)